i18n/es.po

```diff
@@ -62,6 +68,31 @@ msgstr "Tutorial"
 msgid "done"
 msgstr ""
 
+msgid "Export JSON translations"
+msgstr "Exportar traducciones JSON"
+
+msgid "Import JSON translations"
+msgstr "Importar traducciones JSON"
+
+msgid "Close"
+msgstr "Cerrar"
+
+#, fuzzy
```

Comment:
Remove fuzzies, check for the usage and remove `#, fuzzy` if translations seem correct. But for example, I see below _"Customize application text"_ with being translated to _"Installing app"_. The same applies to French. We're (or at least myself) I'm not used to checking translations and I usually let a lot of fuzzies just because I let `yarn localize` do the work and forget to review if there is any fuzzy. But we should normally check. About the translations, now I'm beginning to use AI to generate translations and later do another iteration with AI to spell check.